bin/semver.py

```diff
@@ -156,7 +156,7 @@ def __ge__(self, other):
 
     def __repr__(self):
         s = ", ".join("%s=%r" % (key, val)
-                      for key, val in self._asdict().items())
+                      for key, val in list(self._asdict().items()))
```

Comment:
I don't think listifying this is needed

```suggestion
                      for key, val inself._asdict().items())
```